mm2src/coins/utxo/utxo_common.rs

```diff
@@ -1328,6 +1329,25 @@ pub fn min_trading_vol(coin: &UtxoCoinFields) -> MmNumber {
 
 pub fn is_asset_chain(coin: &UtxoCoinFields) -> bool { coin.conf.asset_chain }
 
+pub async fn get_raw_transaction<T>(coin: T, req: RawTransactionRequest) -> RawTransactionResult
+where
+    T: AsRef<UtxoCoinFields> + UtxoCommonOps + MarketCoinOps,
+{
+    let hash = match H256Json::from_str(&req.tx_hash) {
+        Ok(h) => h,
+        Err(_) => {
+            return MmError::err(RawTransactionError::InvalidHashError(req.tx_hash));
+        },
+    };
+    match coin.as_ref().rpc_client.get_transaction_bytes(hash).compat().await {
+        Ok(hex) => Ok(RawTransactionRes { tx_hex: hex }),
+        Err(err) => {
+            log!("Error: get_transaction_bytes "[err]);
+            return MmError::err(RawTransactionError::HashNotFound(req.tx_hash));
```

Comment:
`get_transaction_bytes` error will not always mean `HashNotFound`, please change this to a more generic `Transport` error or similar.